frontend/ui/data/css_tweaks.lua

```diff
@@ -430,6 +430,13 @@ p {
     margin-bottom: 1em;
 }
             ]],
+        },
+        {
+            id = "cjk_tailored";
+            title = _("Tailor widths and text-indent for CJK"),
+            description = _([[
+Adjust paragraphs' width and text-indent to be an integer multiple of the font size, so that lines of Chinese and Japanese characters don't need to get space added between glyphs by text justification.]]),
```

Comment:
I think I'd prefer just "paragraph width"

"don't need space added between glyphs for text justification"